src/ytdl_sub/prebuilt_presets/tv_show/new_w_enable.yaml

```diff
@@ -0,0 +1,185 @@
+presets:
+
+####################################################################################################
+
+  "Plex":
+    overrides:
+      player: "plex"
```

Comment:
We should add a few helper + validation variables, including

`player_` which reads `player` and validates its either "plex"/"kodi"/"jellyfin"/"emby" (errors otherwise)
`is_kodi_based` - True for kodi/jellyfin/emby, false for plex